analysed-packages/atlassian/version-3.41.3/atlassian-3.41.3-OSS-disclosure.txt

```diff
@@ -0,0 +1,226 @@
+PackageName: site-packages_atlassian-with-distinfo.zip
+PackageVerificationCode: 6f727b7368037c4d5dbbdcbda3b058560582c1f9
+PackageChecksum: SHA256: 014f7aa933dbfa7a7d32f1ef5cf51d1470e221aae9530860039436b2729fc7ef
+PackageLicenseDeclared: Apache-2.0
+--------
+
+FileName: site-packages_atlassian-with-distinfo.zip/site-packages_atlassian-with-distinfo/atlassian_python_api-3.41.3.dist-info/LICENSE:
+FileChecksum: SHA256: c5a9809e85f4cdf05a9888a501f3f79d222dedd22f9e3ee8b9e5f5e25eae28f6
+FileCopyrightText:
+Copyright 2014 Mateusz Harasymczuk, Gonchik Tsymzhitov
+
+LicenseConcluded: LicenseRef-fossology-Apache-2.0-af5b3d00d6efa4c915c087dc57f6ae02

```

Comment:
As already commented in the altgraph PR I do not think that the text of the Apache-2.0 license is licensed under Apache-2.0